code/modules/antagonists/ninja/ninjaDrainAct.dm

```diff
@@ -443,7 +443,10 @@
 
 	force_event(/datum/round_event_control/tram_malfunction, "ninja interference")
 	malfunction_event = locate(/datum/round_event/tram_malfunction) in SSevents.running
-	malfunction_event.end_when *= 3
+	malfunction_event.end_when *= 2
+	for(var/obj/machinery/transport/guideway_sensor/sensor as anything in SStransport.sensors)
+		if(prob(50))
```

Comment:
why prob 50?

Reply:
The actual length of the event is less important now that the engineers just need to fix the controller. I want to strike the right balance of effort (breaking all the components seems too harsh.

I haven't yet decided on what this final number will be, I still want to run some tests with different breakdown values.